33-room-occupancy-monitor/README.md

```diff
@@ -0,0 +1,115 @@
+* [Magnetic Door Switch Set](https://www.sparkfun.com/products/13247)
+
+## Notehub Setup
+
+Sign up for a free account on [notehub.io](https://notehub.io) and [create a new project](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-a/#set-up-notehub).
+
+## Sparrow Setup
+
+### Quickstart
+
+Follow the [Sparrow Quickstart](https://dev.blues.io/quickstart/sparrow-quickstart/) to get your Sparrow reference node paired with the gateway and associated with the Notehub project you just created. Note also that we'll only need one reference node for this project, so you don't need to pair both nodes that came with the dev kit. Make sure that you use the ProductUID generated in [Notehub Setup](#notehub-setup) when it comes time to issue the `hub.set` command in the quickstart.
+
+After you've completed the quickstart, leave the Notecarrier and essentials board powered and connected. These two devices will act as our gateway to Notehub, and we won't need to touch them again. The rest of this guide will focus on the reference node.
+
+### Hardware
+
+There are two primary pieces of hardware: the magnetic door switch and the PIR motion sensor attached to the Sparrow reference node's circuit board.
+
+The magnetic door switch consists of two plastic terminals, one of which has a pair of wires coming out of it. When the terminals are brought into contact (or close proximity), an internal [reed switch](https://en.wikipedia.org/wiki/Reed_switch) is closed and the two wires are electrically connected. As shown in the SparkFun product link above, the terminal with the wires is typically mounted to a door frame and the other terminal is mounted to the door such that when the door is closed, the two terminals are adjacent, closing the switch. Since we're just testing things out, don't mount the terminals yet. Instead, keep them on your work surface and plug the wires into the A1 and GND ports of the Sparrow reference node (it doesn't matter which wire goes into which port). When the switch is closed, A1 will connect to GND. Otherwise, A1 will be driven high by an internal pull-up.
+
+The PIR motion sensor has a small plastic dome sticking out of the front of the reference node's housing. Since this sensor is integrated into the reference node already, there's no additional wiring required. When something moves in front of the sensor, it'll detect that motion and kick off logic in the firmware to send a "motion detected" note to Notehub.
+
+### Firmware
+
+Next, we need to flash the reference node with the room occupancy monitor firmware.
+
+1. Before we do anything, we need to pull in some dependencies for the firmware to work. After cloning this repository, run these commands: `git submodule update --init 33-room-occupancy-monitor/firmware/note-c` and `git submodule update --init 33-room-occupancy-monitor/firmware/sparrow-lora`. This will pull in the note-c and sparrow-lora submodules that this project depends on.
```

Comment:
Do we need two commands? I thought `git submodule update --init` pulls all submodules.

Reply:
@zfields requested it be this way so that you're only pulling in the relevant submodules for this app, rather than all submodules for all apps in the repo.

Reply:
oh yea, good point!